taggit-tacc/taggit
# Task/DES-2411: Taggit Deploy

## Overview: ##
Ported `env.service.ts` and kube configs from hazmapper
Branch deployed to following urls:
https://hazmapper.tacc.utexas.edu/taggit/
https://hazmapper.tacc.utexas.edu/taggit-staging/

## Related Jira tickets: ##

* [DES-2411](https://jira.tacc.utexas.edu/browse/DES-2411)

## Testing Steps: ## 
1. Ensure Jenkins build/deploy works for this branch: task/DES-2411-taggit-deploy
2. Ensure all the endpoints are accessible and taggit works (without the PR'ed fixes not yet merged)
